### ISSUE TITLE
nodejs.org pages in need of translations

### ISSUE THREAD

**Issue by clecorre** [nodejs/nodejs.org#2841]
* **URL**:
* **Browser version**:
* **Operating system**:

Hello, french pages are not well translated, I mean most of the content is not translated at all (see attached, hoping the drag and drop works).
Also, the "change password" page does not work as mentioned. It is written "Make sure it's at least 15 characters OR at least 8 ...". If I enter a 19 characters lowercase string it is refused.
Has anyone ever worked on a french nodejs doc ? Is someone interested in such project  ?

![Untitled](https://user-images.githubusercontent.com/11296904/71099249-fc42fb00-2180-11ea-9cbb-97c2ce247560.png)

**Comment by Trott** [comment]
We rely on volunteers to translate the docs. To help out with the French translations, go to https://crowdin.com/project/nodejs.

**Comment by Trott** [comment]
Oh, but *that* page is not part of the docs. That page would be improved with pull requests to this repository. Sorry about my momentary confusion.

**Comment by Trott** [comment]
To translate that page, clone this repository and create `locale/fr/docs/index.md`. Perhaps copy over `locale/en/docs/index.md` to start and translate it.  I'm sure there are many other pages that need translating into French as well.

**Comment by clecorre** [comment]
Great, thank you.

**Comment by nschonni** [comment]
@Trott CrowdIn isn't setup for the website till https://github.com/nodejs/i18n/issues/72#issuecomment-535355333 is unblocked

**Comment by nschonni** [comment]
@clecorre the Crowdin setup has been enabled if you want to suggest translations now over https://crowdin.com/project/nodejs-website/fr#
